src/index.html

```diff
@@ -4,9 +4,326 @@
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+    <!-- #region main -->
+    <main class="catalog">
+      <ul class="products-list">
+        <li class="products-list__item">
+          <div data-qa="card" class="card card--position">
+            <img
+              src="images/imac.jpeg"
+              alt="APPLE A1419 iMac 27"
+              class="card__image"
+            >
+            <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <p class="card__code">Product code: 195434</p>
+            <div class="card__review-container">
+              <ul class="stars stars--4">
+                <li class="stars__star"></li>
+                <li class="stars__star"></li>
+                <li class="stars__star"></li>
+                <li class="stars__star"></li>
+                <li class="stars__star"></li>
+              </ul>
+              <span class="card__review-count">Reviews: 5</span>
+            </div>
+            <div class="card__price-container">
+              <span class="card__price-label">Price:</span>
+              <span class="card__price-value">$2,199</span>
+            </div>
+            <a
+              data-qa="card-hover"
+              href="#buy"
+              class="card__button"
+            >
+              buy
+            </a>
+          </div>
+        </li>
+
+        <li class="products-list__item">
+          <div class="card">
+            <img
+              src="images/imac.jpeg"
+              alt="APPLE A1419 iMac 27"
+              class="card__image"
+            >
+            <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+            <p class="card__code">Product code: 195434</p>
+            <div class="card__review-container">
+              <ul class="stars stars--4">
+                <li class="stars__star"></li>
+                <li class="stars__star"></li>
+                <li class="stars__star"></li>
+                <li class="stars__star"></li>
+                <li class="stars__star"></li>
+              </ul>
+              <span class="card__review-count">Reviews: 5</span>
+            </div>
+            <div class="card__price-container">
+              <span class="card__price-label">Price:</span>
+              <span class="card__price-value">$2,199</span>
+            </div>
+            <a
+              href="#buy"
+              class="card__button"
+            >
+            buy
+            </a>
```

Comment:
check formatting also
```suggestion
            <a
              href="#buy"
              class="card__button"
            >
              buy
            </a>
```